### PR TITLE
Add an optional third argument for a relative path from the root directory.

### PR DESCRIPTION
**Pull request by eyebrowsoffire**
This is to support .ci.yaml roller in the monorepo. This is to prepare for this recipe change: https://flutter-review.googlesource.com/c/recipes/+/61680

### CODE REVIEW

File: app_dart/test/generate_jspb_test.dart [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  setUp(() {
+    gen_jspb.debugHttpClientFactory = () => mockClient;
+  });
+
+  test('generate with two args', () async {
+    mockClient = MockClient((request) async {
+      expect(request.url, Uri.parse('https://raw.githubusercontent.com/flutter/flutter/flutter/abcde/.ci.yaml'));
+      return http.Response(fakeCiYaml, 200);
+    });
+    await gen_jspb.main(['flutter/flutter', 'abcde']);
```
Comment (christopherfujino) [inline app_dart/test/generate_jspb_test.dart:27]:
```suggestion
    await gen_jspb.main(['flutter', 'abcde']);
```

I think the first arg should just be `flutter`, right? Also see the expected request url, which looks wrong.

Comment (christopherfujino) [reply app_dart/test/generate_jspb_test.dart:27]: this is a nit, since a test.

**Review by christopherfujino — APPROVED** [review]
LGTM

File: app_dart/test/generate_jspb_test.dart (hunk 1 above)
Comment (eyebrowsoffire) [reply app_dart/test/generate_jspb_test.dart:27]: Good catch. I'll change it.

**Pull request merged**